ui/src/testing/stubs/user-admin-api-stub.ts

```diff
@@ -2,13 +2,10 @@ import { AdminUserListResponse, Profile, UserAdminApi } from 'generated/fetch';
 import { ListEgressBypassWindowResponse } from 'generated/fetch';
 
 import { ProfileStubVariables } from './profile-api-stub';
-import { stubNotImplementedError } from './stub-utils';
 
 export class UserAdminApiStub extends UserAdminApi {
   constructor(public profile = ProfileStubVariables.PROFILE_STUB) {
-    super(undefined, undefined, (..._: any[]) => {
-      throw stubNotImplementedError;
-    });
+    super(undefined);
```

Comment:
This ultimately extends BaseApi which only takes one parameter: a configuration object.